Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -10443,8 +10445,15 @@ void WebPageProxy::requestAttachmentIcon(const String& identifier, const String&
     }
 #endif
 
+    String exampleFileName;
```

Comment:
I added Wenson and Tim to confirm that it is indeed OK to only keep the extension part of the filename.

Reply:
Should we initialize to `"example"_s` by default? This way it'd be "example" if we can't find an extension, which seems safer than passing a null String to iconForAttachment().